src/pages/home/home.page.ts

```diff
@@ -54,4 +56,22 @@ export class HomePage {
     }
   }
 
+  ionViewWillLoad() {
+    this.afAuth.authState.subscribe(data =>{
+      if(data && data.email && data.uid){
+        this.toast.create({
```

Comment:
Van egy toastprovider css classal mar hasznald inkabb azt, tobb oldalon is hasznaltam.

Reply:
okes, csak abban nem lehetett create-lni, kijavitom akkor

Reply:
ebben showSuccessMessage createl, s megvan minden amit te hasznalsz itt